keyboards/pk60/info.json

```diff
@@ -26,7 +26,7 @@
+            "layout": [{"label":"~", "x":0, "y":0.25}, {"label":"!", "x":1, "y":0.25}, {"label":"@", "x":2, "y":0.25}, {"label":"#", "x":3, "y":0.25}, {"label":"$", "x":4, "y":0.25}, {"label":"%", "x":5, "y":0.25}, {"label":"^", "x":6, "y":0.25}, {"label":"&", "x":7, "y":0.25}, {"label":"*", "x":8, "y":0.25}, {"label":"(", "x":9, "y":0.25}, {"label":")", "x":10, "y":0.25}, {"label":"_", "x":11, "y":0.25}, {"label":"+", "x":12, "y":0.25}, {"label":"Del", "x":13, "y":0.25}, {"label":"Bs", "x":14, "y":0.25}, {"label":"Tab", "x":0, "y":1.25, "w":1.5}, {"label":"Q", "x":1.5, "y":1.25}, {"label":"W", "x":2.5, "y":1.25}, {"label":"E", "x":3.5, "y":1.25}, {"label":"R", "x":4.5, "y":1.25}, {"label":"T", "x":5.5, "y":1.25}, {"label":"Y", "x":6.5, "y":1.25}, {"label":"U", "x":7.5, "y":1.25}, {"label":"I", "x":8.5, "y":1.25}, {"label":"O", "x":9.5, "y":1.25}, {"label":"P", "x":10.5, "y":1.25}, {"label":"{", "x":11.5, "y":1.25}, {"label":"}", "x":12.5, "y":1.25}, {"label":"|", "x":13.5, "y":1.25, "w":1.5}, {"label":"Caps Lock", "x":0, "y":2.25, "w":1.75}, {"label":"A", "x":1.75, "y":2.25}, {"label":"S", "x":2.75, "y":2.25}, {"label":"D", "x":3.75, "y":2.25}, {"label":"F", "x":4.75, "y":2.25}, {"label":"G", "x":5.75, "y":2.25}, {"label":"H", "x":6.75, "y":2.25}, {"label":"J", "x":7.75, "y":2.25}, {"label":"K", "x":8.75, "y":2.25}, {"label":"L", "x":9.75, "y":2.25}, {"label":":", "x":10.75, "y":2.25}, {"label":"\"", "x":11.75, "y":2.25}, {"label":"\"", "x":12.75, "y":2.25}, {"label":"Enter", "x":13.75, "y":2.25, "w":1.25}, {"label":"Shift", "x":0, "y":3.25}, {"x":1, "y":3.25}, {"label":"Z", "x":2, "y":3.25}, {"label":"X", "x":3, "y":3.25}, {"label":"C", "x":4, "y":3.25}, {"label":"V", "x":5, "y":3.25}, {"label":"B", "x":6, "y":3.25}, {"label":"N", "x":7, "y":3.25}, {"label":"M", "x":8, "y":3.25}, {"label":"<", "x":9, "y":3.25}, {"label":">", "x":10, "y":3.25}, {"label":"?", "x":11, "y":3.25}, {"label":"Shift", "x":12, "y":3.25}, {"label":"Up", "x":13, "y":3.25}, {"x":14, "y":3.25}, {"label":"Ctrl", "x":0, "y":4.25, "w":1.25}, {"label":"Win", "x":1.25, "y":4.25, "w":1.25}, {"label":"Alt", "x":2.5, "y":4.25, "w":1.25}, {"x":3.75, "y":4.25}, {"x":4.75, "y":4.25, "w":4.25}, {"x":9, "y":4.25}, {"label":"Alt", "x":10, "y":4.25}, {"label":"Ctrl", "x":11, "y":4.25}, {"label":"Left", "x":12, "y":4.25}, {"label":"Down", "x":13, "y":4.25}, {"label":"Right", "x":14, "y":4.25}]
```

Comment:
@mechmerlin or @noroadsleft, this look good?

Reply:
This is an improvement.

The only changes are:

1. Some added labels (ignored by the Configurator)
2. All the keys have had their `y` values increased by `0.25`.
3. Key sizes on the bottom row

Item 1 is a non-issue. Some people label these based on what key is commonly placed there (Escape, Enter, Ctrl, etc.); sometimes I name it for the matrix position it has in the keyboard's .h file; sometimes they're blank.

Item 2 is a stylistic annoyance for me, but functionality is unchanged. It actually seems to come from the positions from most of the other layouts, which are offset in a downward direction by 0.25u (compare [LAYOUT_all](https://config.qmk.fm/#/pk60/LAYOUT_all) or [LAYOUT_ansi](https://config.qmk.fm/#/pk60/LAYOUT_ansi) vs. any of the other layouts). For the purposes of this PR, not a deal-breaker.

Item 3, the key sizes on the bottom row aren't 100% accurate when compared against the documentation on the Play Keyboard site (bottom of [this page](http://playkeyboard.qdm.com.tw/60-%E9%8D%B5%E7%9B%A4/PK60)), but it's more accurate than what's on the Configurator now. I could fix it in another PR with probably 15 minutes of work.

If you're okay with merging, I say go ahead.

Reply:
Thanks for explaning. @noroadsleft 

About item 3, it's purpose was just to include all pin outs of board. So not a big deal if keys are not shown in accurate sizes, just for convenient using.

Appreciate for efforts guys.

Reply:
@yj7272098 Whether accurate appearance is helpful to the end users or is just me being obsessed with details is still yet to be determined.

Reply:
Sure :)